### PR TITLE
Readme.MD: PC to BBB USB-serial update

### PR DESCRIPTION
**Pull request by playwithreality**
Improved instructions on how to connect to the BBB via serial.
Updated navigation links and added new section 6.1 for serial troubleshooting.

### CODE REVIEW

**Comment by Arthil** [comment]
There is a commit "Test", which does nothing. It should be removed.

File: README.MD [hunk 1]
```diff
@@ -148,19 +149,16 @@ the setup was also tested to work with Gigabyte NUC, Intel Edison, BeagleBone
 Black and Intel Galileo V2 but they might require some work on the BeagleBone
 filesystem to make them work.
 
-## 2.2 BBB booting
+## 2.2 BBB booting overview
```
Comment (Arthil) [inline README.MD:152]: Remember to change all the old BBB booting links to this new one and also update the table of contents.

**Comment by playwithreality** [comment]
Removed the useless commit "Test". Should be good now.

File: README.MD (hunk 1 above)
Comment (playwithreality) [reply README.MD:152]: Fixed now

**Comment by Arthil** [comment]
Looks good! Test this please.

**Pull request closed without merging**